brands/amenity/police

```diff
@@ -0,0 +1,33 @@
+{
+  "amenity/police|RCMP": {
+    "countryCodes": ["ca"],
+    "tags": {
+      "amenity": "police",
+      "brand": "Royal Canadian Mounted Police",
```

Comment:
Is it really a brand? I would consider switch to `operator`

Reply:
well yes, that is the whole reason this PR has sat since November

Reply:
I know, I wanted to ping author.